requirements.txt

```diff
@@ -11,7 +11,6 @@ Flask==1.0.2
 Flask-OpenID-Stateless==1.2.6
 Flask-WTF==0.14.2
 humanize==0.5.1
-launchpadlib==1.10.7
```

Comment:
:tada:

Reply:
:v: As you can see before we were using anonymous login but it became a nightmare to login with this library